src/components/Footer.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+const Footer = () => {
+    return(
+        <div className = "Footer">
+            <a href = "https://github.com/chingu-voyages/v50-tier1-team-03">Repository Link</a>
```

Comment:
Can you please add a `target` attribute with a value of `"_blank"` so the links will open up in a new tab/window.